### DISCUSSION TITLE
Welcome to Lottery Discussions!

### DISCUSSION THREAD

**Discussion by CI47** [Unqualified-Developers/Lottery#3] [Announcements]
*This is the `Discussions` column of the repository.*
## Vote for New Features
You can set up a vote for `new features` using the template `Ideas` and it will be added to the plan of the project's development if it gets extensive support.
## Ask Questions
If you have some problems about everything of the application, just using the template `Q&A` to ask for help! We will answer your questions if we have time. (We're really very busy.) Don't forget to search for `similar questions` before asking a new question.

**Comment by YBCA7** [comment]
# *Some Important Information*
***We can't make new features at once because the developers are about to take the `university entrance examination`. They will also experience a busy `high school life`. So, it's `difficult` for them to `find spare time` to continue updating this project during this period. However, we promise that we will make them together after several years whenever we have spare time!***
